app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/userModeUI/tools/ai/QuickFixAIChatScreen.kt

```diff
@@ -0,0 +1,153 @@
+package com.arygm.quickfix.ui.uiMode.appContentUI.userModeUI.tools.ai
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.selection.SelectionContainer
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.Send
+import androidx.compose.material.icons.filled.AutoAwesome
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import com.arygm.quickfix.model.tools.ai.GeminiMessageModel
+import com.arygm.quickfix.model.tools.ai.GeminiViewModel
+
+@Preview
+@Composable
+fun QuickFixAIChatScreen(
+    modifier: Modifier = Modifier,
+    viewModel: GeminiViewModel = GeminiViewModel()
+) {
+  Column(modifier = modifier) {
+    MessageList(modifier = Modifier.weight(1f), messageList = viewModel.messageList)
+
+    MessageInput { viewModel.sendMessage(it) }
+  }
+}
+
+@Composable
+fun MessageList(modifier: Modifier = Modifier, messageList: List<GeminiMessageModel>) {
+  if (messageList.size <= 1) {
+    Column(
+        modifier = modifier.fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center) {
+          Icon(
+              imageVector = Icons.Default.AutoAwesome,
+              contentDescription = "Default Background",
+              tint = MaterialTheme.colorScheme.primary,
+              modifier = Modifier.size(230.dp).zIndex(2f))
```

Comment:
We should not use absolute values but seeing the tight deadline with BOTA it's fine